srcpkgs/gnome/template

```diff
@@ -11,69 +11,69 @@ homepage="https://www.gnome.org"
 depends="
  gnome-core>=${version}
  baobab>=${version}
- cheese>=${version}
+ cheese>=41.1
  eog>=${version}
  evince>=${version}
- file-roller>=3.40.0
- gedit>=40.1
+ file-roller>=3.${version}
  gnome-calculator>=${version}
  gnome-calendar>=${version}
  gnome-characters>=${version}
- gnome-clocks>=40.0
+ gnome-clocks>=${version}
  gnome-contacts>=${version}
  gnome-disk-utility>=${version}
  gnome-font-viewer>=${version}
  gnome-maps>=${version}
  gnome-music>=${version}
  gnome-remote-desktop>=${version}
- gnome-screenshot>=40.0
  gnome-system-monitor>=${version}
  gnome-terminal>=3.${version}
- gnome-tweaks>=40.0
+ gnome-text-editor>=${version}
+ gnome-tweaks>=42.beta
  gnome-weather>=${version}
  gsound>=1.0.3
- gspell>=1.9.1
+ gspell>=1.10.0
  mousetweaks>=3.32.0
- network-manager-applet>=1.24.0
  orca>=${version}
  phodav>=2.5
- simple-scan>=40.5
- totem>=3.38.2"
+ simple-scan>=${version}
+ totem>=${version}"
 
 _apps_depends="
  bijiben>=40.1
  dconf-editor>=3.38.3
- devhelp>=${version}
+ devhelp>=41.2
  epiphany>=${version}
- evolution>=3.42.0
- ghex>=3.18.4
- gitg>=3.32.1
- gedit-plugins>=40.1
+ evolution>=3.44.0
+ ghex>=${version}
+ gitg>=41
  gnome-books>=40.0
  gnome-boxes>=${version}
  gnome-builder>=${version}
  gnome-calculator>=${version}
  gnome-calendar>=${version}
+ gnome-connections>=${version}
  gnome-characters>=${version}
- gnome-clocks>=40.0
+ gnome-clocks>=${version}
  gnome-dictionary>=40.0
  gnome-disk-utility>=${version}
  gnome-font-viewer>=${version}
  gnome-maps>=${version}
  gnome-music>=${version}
- gnome-nettool>=3.8.1
- gnome-photos>=40.0
- gnome-recipes>=2.0.2
- gnome-screenshot>=40.0
- gnome-sound-recorder>=40.0
+ gnome-nettool>=${version}
+ gnome-photos>=${version}
+ gnome-recipes>=2.0.4
+ gnome-screenshot>=41.0
+ gnome-sound-recorder>=${version}
  gnome-system-monitor>=${version}
- gnome-terminal>=3.41.0
- gnome-todo>=3.28.1
- gnote>=40.2
+ gnome-terminal>=3.${version}
+ gnome-text-editor>=${version}
+ gnome-todo>=41.0
+ gnote>=${version}
  nemiver>=0.9.6
```

Comment:
the nemiver repository is archived, we should remove it from gnome-apps

Reply:
Alright.